loris/webapp.py

```diff
@@ -621,19 +621,18 @@ def _make_image(self, image_request, src_fp, src_format):
             (str) the fp of the new image
         '''
         # figure out paths, make dirs
-        if self.enable_caching:
-            target_fp = self.img_cache.create_dir_and_return_file_path(image_request)
-        else:
-            # random str
-            n = ''.join(random.choice(string.ascii_lowercase) for x in range(10))
-            target_fp = '%s.%s' % (path.join(self.tmp_dp, n), image_request.format)
+        n = ''.join(random.choice(string.ascii_lowercase) for x in range(10))
+        temp_fp = '%s.%s' % (path.join(self.tmp_dp, n), image_request.format)
```

Comment:
Could use tempfile.NamedTemporaryFile here. You can pass in the tmp_dp as the prefix, and the format as the extension.